Sources/VLCSettingsController.m

```diff
@@ -125,12 +150,46 @@ - (void)updateUIAndCoreSpotlightForPasscodeSetting:(BOOL)passcodeOn
 
 #pragma mark - IASKSettings delegate
 
-- (void)settingsViewController:(IASKAppSettingsViewController*)sender buttonTappedForSpecifier:(IASKSpecifier*)specifier
+- (IBAction)pushAboutViewController:(id)sender
 {
     VLCAboutViewController *aboutVC = [[VLCAboutViewController alloc] init];
     [self.navigationController pushViewController:aboutVC animated:YES];
 }
 
+- (CGFloat) settingsViewController:(id<IASKViewController>)settingsViewController tableView:(UITableView *)tableView heightForHeaderForSection:(NSInteger)section {
+    return section == 0 ? 0 : SETTINGS_HEADER_HEIGHT;
+}
+
+- (UIView *)settingsViewController:(id<IASKViewController>)settingsViewController tableView:(UITableView *)tableView viewForHeaderForSection:(NSInteger)section
+{
+    UIView *header = [[UIView alloc] initWithFrame:CGRectMake(HEADER_OFFSET, 0, tableView.bounds.size.width, SETTINGS_HEADER_HEIGHT)];
```

Comment:
Why are you not using a UITableViewHeaderFooterView? I know that this is only a tiny class but I'd still rather have that as it's own class